src/components/MovieListPage/MovieListPage.js

```diff
@@ -0,0 +1,89 @@
+import React, { useState, useEffect } from 'react';
+import './MovieListPage.scss';
+import { GENRE_LIST, MOVIES } from "../../App";
+import MovieTile from "../MovieTile/MovieTile";
+import MovieDetails from "../MovieDetails/MovieDetails";
+import SearchForm from "../SearchForm/SearchForm";
+import GenreList from "../GenreList/GenreList";
+import SortControl from "../SortControl/SortControl";
+import axios from 'axios';
+
+const MovieListPage = () => {
+    const [searchQuery, setSearchQuery] = useState('');
+    const [sortByOptions, setSortByOptions] = useState([
+        {
+            label: 'Release Date', value: 'releaseDate',
+        },
+        {
+            label: 'Title', value: 'title',
+        }
+    ]);
+    const [sortCriterion, setSortCriterion] = useState([sortByOptions[0]]);
+    const [activeGenre, setActiveGenre] = useState('');
+    const [movieList, setMovieList] = useState(MOVIES);
+    const [selectedMovie, setSelectedMovie] = useState(null);
+
+    useEffect(() => {
+        async function getData() {
+            debugger;
+            const data = await axios.get(
```

Comment:
Create a seperate `api.js` file, where you will export axios with basing configuration (base url). 

You will use it in multiple places so it will be more handly to append endpoint only.

https://axios-http.com/docs/config_defaults